fix_proto_conf_desc.py

```diff
@@ -0,0 +1,177 @@
+import ida_segment
+import idautils
+import idc
+import ida_bytes
+import ida_name
+from ida_nalt import REF_OFF32, REFINFO_SIGNEDOP, REFINFO_SELFREF
+from idc import FF_DATA, FF_DWORD
+
+class CStruct: 
+    tif = 0
```

Comment:
this makes it so `tif` is a static variable which should not be the desired behavior as we discussed. instead put it in an `__init__` method so this class will have an object meaning, currently you don't really use `self`